packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -688,7 +690,9 @@ class DebuggerController extends DisposableController
   /// We call this method as we expand variables in the variable tree, because
   /// building the tree for all variable data at once is very expensive.
   Future<void> buildVariablesTree(Variable variable) async {
-    if (!variable.isExpandable || variable.treeInitialized) return;
+    if (!variable.isExpandable ||
```

Comment:
The addition of the entires clause means that there can be an expandable Variable without an instance ref. In this case, child variables are already added so this needs to short-circuit.